### PR TITLE
time-picker min max logic fixed

### PR DESCRIPTION
**Pull request by enchev**

### CODE REVIEW

**Comment by enchev** [comment]
run ci

**Comment by hamorphis** [comment]
:+1:

**Comment by vakrilov** [comment]
:+1:

File: ui/time-picker/time-picker-common.ts [hunk 1]
```diff
@@ -5,59 +5,138 @@ import view = require("ui/core/view");
...
-        new proxy.PropertyMetadata(undefined, dependencyObservable.PropertyMetadataSettings.None, undefined, isHourValid));
+        new proxy.PropertyMetadata(0, dependencyObservable.PropertyMetadataSettings.None, onHourPropertyChanged, isHourValid));
 
     public static minHourProperty = new dependencyObservable.Property("minHour", "TimePicker",
-        new proxy.PropertyMetadata(1, dependencyObservable.PropertyMetadataSettings.None, undefined, isHourValid));
+        new proxy.PropertyMetadata(0, dependencyObservable.PropertyMetadataSettings.None, onMinHourPropertyChanged, isHourValid));
 
     public static maxHourProperty = new dependencyObservable.Property("maxHour", "TimePicker",
-        new proxy.PropertyMetadata(24, dependencyObservable.PropertyMetadataSettings.None, undefined, isHourValid));
+        new proxy.PropertyMetadata(23, dependencyObservable.PropertyMetadataSettings.None, onMaxHourPropertyChanged, isHourValid));
```
Comment (vakrilov) [inline ui/time-picker/time-picker-common.ts]: :+1:

**Pull request merged**